src/lib.rs

```diff
@@ -1,3 +1,8 @@
+#![cfg_attr(
+    all(target_os = "windows", feature = "nightly"),
+    feature(windows_by_handle)
+)]
```

Comment:
I added those lines just to enable the `windows_by_handle` feature if we are on windows AND using the nightly rust toolchain.

Reply:
I understand, it makes sense. I will remove the guard!

Reply:
:heavy_check_mark: